src/CSET/operators/plot.py

```diff
@@ -429,6 +429,131 @@ def _plot_and_save_vertical_line_series(
     plt.close(fig)
 
 
+def _plot_and_save_histogram_series(
+    cube: iris.cube.Cube, filename: str, title: str, vmin: float, vmax: float, **kwargs
+):
+    """Plot and save a histogram series.
+
+    Parameters
+    ----------
+    cube: Cube
+        2 dimensional Cube of the data to plot as histogram.
+        Plotting options are fixed:
+        density=True, histtype='bar',stacked=True to ensure that
+        a probability density is plotted using matplotlib.pyplot.hist
+        to plot the probability density so that the area under
+        the histogram integrates to 1.
+        stacked is set to True so the sum of the histograms is
+        normalized to 1.
+        ax.autoscale is switched off and the ylim range
+        is preset as (0,1) to make figures comparable.
+    filename: str
+        Filename of the plot to write.
+    title: str
+        Plot title.
+    vmin: float
+        minimum for colourbar
+    vmax: float
+        maximum for colourbar
+    """
+    fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
+    # reshape cube data into a single array to allow for a single histogram.
+    # Otherwise we plot xdim histograms stacked.
+    cube_data_1d = (cube.data).flatten()
+    plt.hist(cube_data_1d, density=True, histtype="bar", stacked=True)
+    ax = plt.gca()
+    # try seaborn for a box whisker plot:
+    # https://python-graph-gallery.com/34-grouped-boxplot/
```

Comment:
```suggestion

```
Suggesting to remove as not sure what this comment is referring to

Reply:
thanks removed in 8ccc6d9..fd08bc0